openchallenges/data_integration_lambda.py

```diff
@@ -0,0 +1,38 @@
+from aws_cdk import aws_iam as iam
+from aws_cdk import aws_lambda as _lambda
```

Comment:
Is there any non-private (ie: Something not prefixed by an `_`) that can be used here. Since you cannot have private functions in python this is used to denote that users of the library shouldn't really be using them directly.

If it's not available elsewhere, then that's still ok. We will just need to be careful on upgrades of the library as they're more likely to break on version updates.

Reply:
I replaced `_lambda` by `lambda_`, which [seems](https://stackoverflow.com/a/21075256) to be a preferred way to prevent conflicts with Python keywords.